samples/subsys/fs/fat_fs/sample.yaml

```diff
@@ -14,7 +14,7 @@ tests:
     platform_allow: nrf52840_blip
   sample.filesystem.fat_fs.adafruit_2_8_tft_touch_v2:
     depends_on: arduino_spi arduino_gpio arduino_i2c
-    platform_exclude: stm32f769i_disco pan1781_evb pan1782_evb frdm_k64f
+    platform_exclude: stm32f769i_disco pan1781_evb pan1782_evb frdm_k64f mimxrt1010_evk
```

Comment:
WHy is fat_fs sample excluded?

Reply:
The sample requires the `adafruit_2_8_tft_touch_v2` shield and this shield requires an special Arduino header signal for the SPI CS (AFAICR signal D10). Sadly this signal is not connected from SOC to the header in factory default on this board. So the `gpio-map` will not provide the needed signal label for the Adafruit display shield and the build will fail. That's why this board exclusion is here (and all the others).

Reply:
@mmahadevan108 : Does that answer your question? If so, I'd like to close this conversation.